### ISSUE TITLE
error thrown in mobile safari

### ISSUE THREAD

**Issue by ohmycode** [danigb/tonal#23]
Hey @danigb,

I just noticed that when I try to run your library from mobile Safari, it throws an error:

"SyntaxError: Unexpected keyword 'const'. Const declarations are not supported in strict mode."

I don't know if the problem is iOS (9.3.5) not supporting ES6 or if strict mode should't be specified in each of your modules.

**Comment by warriorpostman** [comment]
@ohmycode In order to run ES6-based code on older browsers, I believe you need to accomodate backwards-compatibility using Babel, es6-shim or something similar with your web-app code.

**Comment by ohmycode** [comment]
yes, thank you @warriorpostman, that's what I've com up with also. 
For further reference: [babel-plugin-transform-es2015-block-scoping](https://www.npmjs.com/package/babel-plugin-transform-es2015-block-scoping) is the babel plugin that converts const ant let back to var.

**Comment by danigb** [comment]
Hi,

Thanks @ohmycode for reporting and @warriorpostman for the answer 👍 👏  

... but, I tried to make tonal ES5 compatible out-of-the-box providing `/dist` files for all packages. @ohmycode, can you give me a snippet of code to reproduce the issue?

Thanks both!

**Comment by ohmycode** [comment]
Hey @danigb,

I've included tonal in a [meteor](https://www.meteor.com/) package as a NPM dependency.

`Npm.depends({
    tonal : '0.69.5'
});`

In that case meteor takes care of the building process, which uses Babel as compiler. I guess It would be a bit cumbersome to include a packed file out of the dist folder instead of using the NPM package. Actually I didn't managed to get the babel-plugin-transform-es2015-block-scoping plugin working in my meteor setup, so I simply removed all occurrences of `'use strict;'` in the compiled file as a quick fix. There is also a [ES5-Shim](https://atmospherejs.com/meteor/es5-shim) package for meteor which I haven't tested.
